zookeeper-it/pom.xml

```
@@ -33,6 +33,16 @@
     ZooKeeper system tests
   </description>
 
+  <profiles>
+    <profile>
+      <id>full-build</id>
+      <properties>
+         <skipTests>true</skipTests>
+         <spotbugs.skip>true</spotbugs.skip>
```

Lesson learned from #1314 : you might need to skip checkstyle, too.

```suggestion
         <checkstyle.skip>true</checkstyle.skip>
         <skipTests>true</skipTests>
         <spotbugs.skip>true</spotbugs.skip>
```